src/keyring/KeyringController.ts

```diff
@@ -285,6 +304,8 @@ export class KeyringController extends BaseController<KeyringConfig, KeyringStat
         }
         privateKey = ethUtil.bufferToHex(wallet.getPrivateKey());
         break;
+      default:
+        throw new Error(`Unexpected import strategy: '${strategy}'`);
```

Comment:
This is a functional change. Previously it would successfully add an _empty_ simple keyring, accomplishing nothing.